### PR TITLE
Update UUIDs of Services

### PR DESCRIPTION
**Pull request by openBackhaul**
**Process for updating the UUIDs of Services**
- [x] Thorsten to check categories of OperationServers and OperationClients in ServiceList
- [x] Iswaryaa to review categories of OperationServers and OperationClients in ServiceList
- [x] Katharina to add new UUIDs based on category of operation (but not old UUID) in ServiceList
- [x] Iswaryaa to review new UUIDs in ServiceList
- [x] Katharina to CTRL+h for old UUIDs and replace them by new UUIDs in ForwardingList, CONFIGfile and OAS.
- [x] Katharina to step through ForwardingList, CONFIGfile and OAS and compare every UUID against new one in ServiceList
- [x] Iswaryaa to review new UUIDs in ForwardingList, CONFIGfile and OAS
If no errors found
- [x] Iswaryaa to delete old UUIDs in ServiceList
- [x] Iswaryaa to merge the feature branch into develop
- [x] Iswaryaa to close this issue and delete the feature branch on GitHub

### CODE REVIEW

**Comment by IswaryaaS** [comment]
services.yaml file looks good for new uuid updates. We can proceed updating other files new uuid(s) now

**Comment by kmohr-soprasteria** [comment]
updated uuids according to mapping table provided in issue:
- config file: no entries related to no longer used found
- forwardings file: no updates
- oas: updates added, no entries related to no longer used found

**Comment by kmohr-soprasteria** [comment]
no changes required in fowardings; mapping table attached to issue #71

File: ExecutionAndTraceLog+services.yaml [hunk 1]
```diff
@@ -1,280 +1,8204 @@
+openapi: 3.0.0
+info:
+  title: ExecutionAndTraceLog
```
Comment (IswaryaaS) [inline ExecutionAndTraceLog+services.yaml:3]: Somehow this file got replaced by ExecutionAndTraceLog.yaml . Requesting you to check and replace it with service.yaml file

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: ExecutionAndTraceLog+services.yaml (hunk 1 above)
Comment (kmohr-soprasteria) [reply ExecutionAndTraceLog+services.yaml]: restored the correct file contents, ready for review

**Pull request merged**